File: test/fixtures/sources/hello.py
class Greeter:
    def __init__(self, name: str) -> None:
        self.name = name

    def greet(self) -> str:
        return f"Hello, {self.name}!"


def main() -> None:
    print(Greeter("world").greet())


if __name__ == "__main__":
    main()

File: test/fixtures/sources/hello.rs
pub struct Greeter {
    name: String,
}

impl Greeter {
    pub fn new(name: &str) -> Self {
        Self { name: name.to_string() }
    }

    pub fn greet(&self) -> String {
        format!("Hello, {}!", self.name)
    }
}

fn main() {
    let g = Greeter::new("world");
    println!("{}", g.greet());
}
